src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+    private String color = getSupplier().getRandomColor();
```

Comment:
```suggestion
```

Reply:
not fixed: 

you should generate color in method, not in object